src/pharmpy/modeling/basic_models.py

```diff
@@ -95,7 +105,11 @@ def create_basic_pk_model(
     vc_ass = Assignment(VC, pop_vc.symbol * sympy.exp(sympy.Symbol(eta_vc_name)))
 
     cb = CompartmentalSystemBuilder()
-    central = Compartment.create('CENTRAL', doses=(dosing(di, df, 1),))
+    # FIXME: This shouldn't be used here
+    from pharmpy.model.external.nonmem.advan import dosing, find_dose
```

Comment:
As the FIXME says. The nonmem specific function should not be used here. It assumes that this is a nonmem dataset.